src/display/api.js

```diff
@@ -1497,6 +1503,7 @@ class PDFPageProxy {
       // Only include the required properties, and *not* the entire object.
       params: {
         canvasContext,
```

Comment:
I think this can be removed now that `InternalRenderTask` seems to no longer use this given that this patch switched it to use `canvas` instead?